SH4RE/SH4RE/Components/ImageCarousel.swift

```diff
@@ -14,30 +14,24 @@ struct ImageCarouselView<Content: View>: View {
     private var content: Content
     @State private var currentIndex: Int = 0
     @State private var offset = CGSize.zero
+    @State private var show:Bool = true;
 
     init(numberOfImages: Int, isEditable: Bool, @ViewBuilder content: () -> Content) {
         self.numberOfImages = numberOfImages
         self.isEditable = isEditable
         self.content = content()
     }
 
-    var body: some View {
-        if (currentIndex != numberOfImages - 1 && isEditable) {
-            Button(action: {
-                self.deleteImage!(currentIndex)
-            }) {
-                ZStack {
-                    Circle()
-                        .fill(Color.primaryDark)
-                        .frame(width: 25, height: 25)
-                    Image(systemName: "xmark")
-                        .font(.system(size: 15, weight: .bold, design: .rounded))
-                        .foregroundColor(.white)
-                }
-                .padding(8)
-                .contentShape(Circle())
-            }
+    func animateCloseButton () {
```

Comment:
this function is to make the X button disspear and reappear when images are slid. Otherwise, the X button stays static while images slide and it looks really weird. I tried making X button slide with image but its quite difficult so this seemed like an easier workaround

Reply:
👍